### PR TITLE
[RFR] Bootstrap Builder + Add SelectOne Query

### PR DESCRIPTION
**Pull request by jdemangeon**

### CODE REVIEW

File: packages/postgres-queries-builder/src/queries/index.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export * from './select';
+// export * from './insert';
```
Comment (Kmaschta) [inline packages/postgres-queries-builder/src/queries/index.ts]: Commented code

Comment (jdemangeon) [reply packages/postgres-queries-builder/src/queries/index.ts]: Normal

Comment (alexisjanvier) [reply packages/postgres-queries-builder/src/queries/index.ts]: Why ?

File: packages/postgres-queries-builder/src/configuration.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export interface Query {
+    sql: string;
+    parameters: StringMap;
+    returnOne: boolean;
+}
+
+export type QueryFun = (raw: StringMap | string) => Query;
+
+export default interface Config {
```
Comment (alexisjanvier) [inline packages/postgres-queries-builder/src/configuration.ts]: Do we decide not to export by Default?
And what is this file for?

File: packages/postgres-queries-builder/src/queries/index.ts (hunk 1 above)
Comment (jdemangeon) [reply packages/postgres-queries-builder/src/queries/index.ts]: I can remove if you want but it's going to be uncommented

File: packages/postgres-queries-builder/src/helpers/sanitizeIdentifier.spec.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import sanitizeIdentifier from './sanitizeIdentifier';
```
Comment (alexisjanvier) [inline packages/postgres-queries-builder/src/helpers/sanitizeIdentifier.spec.ts]: Are test files named `.test.` or `.spec.`?
Also, I'm in favour of adopting [KebabCase](http://wiki.c2.com/?KebabCase) for file names.

**Comment by jdemangeon** [comment]
Review applied

**Pull request merged**